README.md

```diff
@@ -1 +1,64 @@
-# Module1-hellomicroverse
\ No newline at end of file
+# Hello Microverse
+
+> Hello Microverse is first project for microverse .
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](http://127.0.0.1:5500/index.html)
+
+## Getting started
+
+Go to the Folder where want to copy this repo and type the following cmds in you terminal:
+
+- git clone git@github.com:HenaryKhuraijam/Module1-hellomicroverse.git
+
+## Prerequisites
+
+- Knowledge HTML and CSS
+- VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Node package. [Link to download node](https://nodejs.org/en/download/)
+
+## To Install type the following command
+
+- npm install
+
+## To Run App type the following command in your terminal
+
+  - npm start
+
+### To Run test cases type the following command
+
+  - npm test
+
+## Author
+
+👤 Khuraijam Henary Singh
+
+- GitHub: [@HenaryKhuraijam](https://github.com/HenaryKhuraijam)
+- Twitter: [@HenaryKhuraijam](https://twitter.com/HenaryKhuraijam)
+- LinkedIn: [Henary Khuraijam](https://www.linkedin.com/in/henary-khuraijam)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues).
+
+## Show your support
+
+Give a [⭐️](../../stargazers) if you like this project!
+
+
+## Acknowledgments
+- Microverse Team
+- My Family
+- All my fellow micronauts.
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] **Good job on using the MICROVERSE Readme template 👍** However,

    I think it would be better to provide a  valid **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator :joy: 

    _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

Reply:
Thank you for your suggestion, I will make the required changes.